internal/servers/controller/handlers/hosts/host_service.go

```diff
@@ -265,6 +257,7 @@ func (s Service) UpdateHost(ctx context.Context, req *pbs.UpdateHostRequest) (*p
 		idActions := idActionsTypeMap[host.SubtypeFromId(req.GetId())]
 		outputOpts = append(outputOpts, handlers.WithAuthorizedActions(authResults.FetchActionSetForId(ctx, h.GetPublicId(), idActions).Strings()))
 	}
+	outputOpts = append(outputOpts, handlers.WithHostSetIds(h.GetSetIds()))
```

Comment:
I didn't realize the UpdateHost call didn't already return set ids for plugin based hosts.  Since this was something that didn't exist before can you add the test for TestUpdate_Plugin as well (and update the update host flow in the plugin repo if needed)?

Reply:
Woops! Plugin doesn't have "update" so my comment doesn't make sense.